checker/check-request-property-default-value-changed.go

```diff
@@ -0,0 +1,64 @@
+package checker
+
+import (
+	"fmt"
+
+	"github.com/tufin/oasdiff/diff"
+)
+
+func RequestPropertyDefaultValueChangedCheck(diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, config Config) Changes {
+	result := make(Changes, 0)
+	if diffReport.PathsDiff == nil {
+		return result
+	}
+	for path, pathItem := range diffReport.PathsDiff.Modified {
+		if pathItem.OperationsDiff == nil {
+			continue
+		}
+		for operation, operationItem := range pathItem.OperationsDiff.Modified {
+			if operationItem.RequestBodyDiff == nil ||
+				operationItem.RequestBodyDiff.ContentDiff == nil ||
+				operationItem.RequestBodyDiff.ContentDiff.MediaTypeModified == nil {
+				continue
+			}
+			source := (*operationsSources)[operationItem.Revision]
+
+			modifiedMediaTypes := operationItem.RequestBodyDiff.ContentDiff.MediaTypeModified
+			for mediaType, mediaTypeDiff := range modifiedMediaTypes {
+				if mediaTypeDiff.SchemaDiff != nil && mediaTypeDiff.SchemaDiff.DefaultDiff != nil {
+					defaultValueDiff := mediaTypeDiff.SchemaDiff.DefaultDiff
+					result = append(result, ApiChange{
+						Id:          "request-body-default-value-changed",
+						Level:       INFO,
+						Text:        fmt.Sprintf(config.i18n("request-body-default-value-changed"), ColorizedValue(mediaType), empty2none(defaultValueDiff.From), empty2none(defaultValueDiff.To)),
+						Operation:   operation,
+						OperationId: operationItem.Revision.OperationID,
+						Path:        path,
+						Source:      source,
+					})
+				}
+
+				CheckModifiedPropertiesDiff(
+					mediaTypeDiff.SchemaDiff,
+					func(propertyPath string, propertyName string, propertyDiff *diff.SchemaDiff, parent *diff.SchemaDiff) {
+						if propertyDiff == nil || propertyDiff.Revision == nil || propertyDiff.Revision.Value == nil || propertyDiff.DefaultDiff == nil {
```

Comment:
1. propertyDiff.Revision.Value is not used below, so no need to check it
2. other checkers assume that Base and Revision fields are not nil, I think we can do the same here (for consistency)

Reply:
this was copied from check-response-property check which had a fix related to a nil pointer https://github.com/Tufin/oasdiff/pull/272

Reply:
ah! you're right, removed! thanks. also, I don't have permissions to merge the PR even with your approval